docs/beanstalk/02_Creating_EB_Environment/README.md

```diff
@@ -0,0 +1,88 @@
+# Creating Your Elastic Beanstalk Environment
+
+Now that you have the EB CLI  installed, you will be able to push your Backend (Server) code to Elastic Beanstalk in your AWS account. 
+
+Note that all of the following will take place withing the "server" directory.
+
+## Initializing Your Elastic Beanstalk Project
+
+Initializing your Elastic Beanstalk project will create a `.elasticbeanstalk` folder which will contain a configuration file with information about your EB deployment.
+
+
+<br>
+1) Run the "eb init" command which will prompt you for a few questions. You make your selection by typing the matching number or character and pressing "Enter".  
+ Make sure to select the "US East (N. Virginia)" option which should be #1.
+
+![Running "eb init" command and selecting Option 1 <US East N. Virginia>)](./images/EB_CLI_INIT_1_Select_Region.png)
+
+
+<br>
+2) You will now be prompted to either create an application or select an existing on. Enter the number for "Create a new Application"
+
+![Selecting "Create a new Application" option](./images/EB_CLI_INIT_2_Create_New_App.png)
+
+
+<br>
+3) Next you will be prompted to enter the name of your project. In this example "CSCL" was used. Feel free to use a different name.
+
+![Entering the name for our EB application](./images/EB_CLI_INIT_3_Set_Name.png)
+
+
+<br>
+4) Now you will be asked if you are "using Node.js", enter 'y` for "yes". Next, choose the "Node.js 12" option which should be #1.
+
+![Enter "y" to confirm that you are using "Node.js" and select "Node.js 12"](./images/EB_CLI_INIT_4_Configure_Platform.png)
+
+
+<br>
+5)  You will be prompted to create an "SSH key pair". Enter 'y' to accept. For the name enter "cscl", it must be that name as it will be referenced during the deployment.
+You can ignore the warning about 
+
+![Enter 'y' to create an SSH key pair and enter the name "cscl"](./images/EB_CLI_INIT_5_SSH_Key_Setup.png)
+
+
+<br>
+6) You will be prompted to enter a passphrase. Press `Enter` two times to enter a <Blank> passphrase if you don't wish to use one. Setting a passphrase will require you to enter it whenever you attempt to connect to your server using SSH.
```

Comment:
I would keep "Enter" consistent with what's on line 13. And I think <Blank> is getting interpreted like an HTML tag so you might be fine with just saying "a blank passphrase".